bot/exts/filters/webhook_remover.py

```diff
@@ -72,7 +75,12 @@ async def on_message(self, msg: Message) -> None:
 
         matches = WEBHOOK_URL_RE.search(msg.content)
         if matches:
-            await self.delete_and_respond(msg, matches[1] + "xxx")
+            async with self.bot.http_session.delete(msg.content) as resp:
+                # The Discord API Returns a 204 NO CONTENT repsonse on success.
+                if resp.status == 204:
+                    await self.delete_and_respond(msg, matches[1] + "xxx", True)
+                else:
+                    await self.delete_and_respond(msg, matches[1] + "xxx", False)
```

Comment:
The connection doesn't need to be held open to send the discord message, we can also get rid of the conditional
```suggestion
            async with self.bot.http_session.delete(msg.content) as resp:
                # The Discord API Returns a 204 NO CONTENT repsonse on success.
                deleted_successfully = resp.status == 204
            await self.delete_and_respond(msg, matches[1] + "xxx", deleted_successfully)
```

Reply:
Valid point. Very inefficient, it is.